### ISSUE TITLE
404 error

### ISSUE THREAD

**Issue by srakl** [pistej/yii2-faq#1]
Hi
did the migration

`php yii migrate/up --migrationPath=vendor/pistej/yii2-faq/migrations/`

and added this in module

```
'modules' => [
    ...
    'faq' => [
        'class' => 'pistej\faq\Faq',
    ],
],
```

but when i go to
`localhost/frontend/web/faq/qa/index` or
`localhost/frontend/web/faq/group/index` 

i get a 404 error, any idea what i'm doing wrong here?

also where do i put this?

`<?= \pistej\faq\widgets\FaqWidget\FaqWidget::widget(); ?>`

**Comment by pistej** [comment]
Hi,
not sure what app structure you use, advanced? ... configuration register new module, so if its in your frontend config, URL should be fine.

`<?= \pistej\faq\widgets\FaqWidget\FaqWidget::widget(); ?>` put it in ...view/layouts/main.php  into `<body><main>` section to be loaded on each page.

**Comment by srakl** [comment]
yes, i'm using the advanced structure. i'm still getting a 404 error. do i need to create a faq module in my frontend with controllers qa and group with actions index?
